StopSDL.sh

```diff
@@ -15,7 +15,7 @@ function shutdown_sdl {
 
 function kill_sdl {
     echo "Kill SDL process : "$pid
-    kill -SIGTERM $pid
+    kill -SIGABRT $pid
```

Comment:
@orhan-mehmedov ,  it can be SIGKILL too, I changed from SIGTERM, because SDL catch this signal